### ISSUE TITLE
is ZGC stable

### ISSUE THREAD

**Issue by quhaiyang** [alibaba/dragonwell11#23]
is ZGC stable?
do you recommend using ZGC in production environment?

wish your response~

**Comment by luchsh** [comment]
> is ZGC stable?
> do you recommend using ZGC in production environment?
> 
> wish your response~

Nope, personally I would not recommend ZGC for production use with Dragonwell 11, 
neither with any other OpenJDK11 distributions. 
Unless someone backports those production patches from tip.
> FYI, this is not an official reply, since I'm not working for Alibaba anymore.

**Comment by luchsh** [comment]
@tanghaoth90  is there a plan now to provide a production-ready ZGC for Dragonwell 11?

**Comment by tanghaoth90** [comment]
@luchsh Most features of production-ready ZGC are backported to our in-house JDK implementation which has been used by three internal customers. We are fixing and polishing the implementation according to their feedback and the upstream jtreg tests. After that, we might release a separate internal version to reach out for more customers.
